### PR TITLE
Reflect recent changes and gained experience (nginx-ingress-* charts, ES outage on-site)

### PR DESCRIPTION
**Pull request by lucendio**
* migration from `nginx-lb-ingress` to `nginx-ingress-controller` & `nginx-ingress-services`
* note on trouble shoot ES in a situation where disk ran full
* some minor fixes I discovered back when I started and tried to deploy for the first time

### CODE REVIEW

File: src/how-to/administrate/elasticsearch.rst [hunk 1]
```diff
@@ -57,3 +57,14 @@ How to manually look into what is stored in elasticsearch
...
 See also the elasticsearch sections in :ref:`investigative_tasks`.
+
+Troubleshooting
+~~~~~~~~~~~~~~~
+
+Description:
+**ES nodes ran out of disk space** and error message says: ``"blocked by: [FORBIDDEN/12/index read-only / allow delete (api)];"``
+
+Solution:
+* clean up disk (e.g. ``apt autoremove`` on all nodes) & restart nodes a/o ES process
```
Comment (jschaul) [inline src/how-to/administrate/elasticsearch.rst]:
```suggestion
* clean up disk (e.g. ``apt autoremove`` on all nodes), then restart machines and/or the elasticsearch process
```

File: src/how-to/administrate/elasticsearch.rst [hunk 2]
```diff
@@ -57,3 +57,14 @@ How to manually look into what is stored in elasticsearch
...
+* get ES out of *read-only* mode: ``curl -X PUT -H 'Content-Type: application/json' https://${ES_HOST}:${ES_PORT}/_all/_settings -d '{"index.blocks.read_only_allow_delete": null}'``
```
Comment (jschaul) [inline src/how-to/administrate/elasticsearch.rst]: For consistency with the rest of the page, as "ES_HOST/ES_PORT" are not defined.

```suggestion
* get the elastichsearch cluster out of *read-only* mode: SSH to one elasticsearch machine, then run ``curl -X PUT -H 'Content-Type: application/json' http://localhost:9200/_all/_settings -d '{"index.blocks.read_only_allow_delete": null}'``
```

File: src/how-to/administrate/elasticsearch.rst [hunk 3]
```diff
@@ -57,3 +57,14 @@ How to manually look into what is stored in elasticsearch
...
+* trigger reindexing: ``curl -v -X POST $BRIG_IP:8080/i/index/reindex``
```
Comment (jschaul) [inline src/how-to/administrate/elasticsearch.rst]:
```suggestion
* trigger reindexing: From a kubernetes machine, one one terminal port-forward to into brig useing ``kubectl port-forward svc/brig 9999:8080``, and in a second terminal trigger the reindex: ``curl -v -X POST localhost:9999/i/index/reindex``
```

Comment (lucendio) [reply src/how-to/administrate/elasticsearch.rst]: > *...port-forward to into brig...*

Not sure, but it feels a bit hard to read. maybe *to* is unnecessary here?

Comment (jschaul) [reply src/how-to/administrate/elasticsearch.rst]: Ah sorry, multiple typos in my sentence, I updated the suggestion.

**Pull request merged**